src/gettingstarted/dev-environments/merge.md

```diff
@@ -0,0 +1,50 @@
+# Development Environments
+
+## Merge into production
+
+Now that you've had the chance to verify that your application built and deployed correctly on your development environment, you're ready to merge it into your production site. Platform.sh provides [snapshot](/administration/snapshot-and-restore.md) features that protect against any unforeseen consequences of your merges, keeping a historical copy of all of your code and data.
+
+<asciinema-player src="/videos/asciinema/snap-merge-restore.cast" preload=1 autoplay=1 loop=1></asciinema-player>
+
+> **Note:** The `--project` flag is not needed if you are merging from within your local repository.
+
+1. **Create a snapshot**
+
+    Before you merge the `dev` feature into `master`, create a [snapshot](/administration/snapshot-and-restore.md) of the `master` environment. The snapshot will preserve both the code and all of its data.
+
+    ```bash
+    platform snapshot:create --project <project id>
+    ```
+
+    Select `master` as the environment you want to back up.
+
+2. **Merge feature into production**
+
+    ```bash
+    git checkout master
+    platform merge --project <project id>
+    ```
+
+    Select `dev`, and then Platform.sh will merge `dev` into its parent, `master`. When the build process completes, verify that your changes have been merged.
```

Comment:
This is one where we definitely shouldn't prompt and should just use command line switches.  Or just use Git itself directly.  I don't think I've ever used the `platform` command to merge, ever, in 3 years.